test/unit/host_test.rb

```diff
@@ -1907,7 +1907,32 @@ def to_managed!
     assert host.errors[:provision_method].include?("can't be updated after host is provisioned")
   end
 
-  test "#image_build? must be true when provision_method is image" do
+  test "#provision_methods must include build and image by default" do
+    host = FactoryGirl.create(:host, :managed)
```

Comment:
prefer `.build` rather than `.create` as it'll be instantiated and not saved to the database, speeding up the test